lib/utilities/hardware.js

```diff
@@ -25,3 +26,38 @@ export async function getxpub(options) {
     path,
   };
 }
+
+/*
+ * get signature from hardware wallet device
+ * @param {Object} options
+ */
+export async function getHardwareSignatures(options) {
+  const { hardwareType, proposalTX, chain, network } = options;
+  let { path, account } = options;
+
+  if (typeof account === 'string') account = parseInt(account, 10);
+
+  if (!path) {
```

Comment:
This code was copy/pasted from the function above, should refactor it into a function private to this module